src/new-client/src/App/LiveRates/Tile/HistoricalGraph/HistoricalGraph.tsx

```diff
@@ -97,3 +97,59 @@ export const HistoricalGraph: React.FC<HistoricalGraphProps> = ({
     </LineChartWrapper>
   )
 }
+
+export const HistoricalGraphComponent: React.FC<{
+  showTimer: any
+  history: any
+  active: boolean
+  isAtom?: boolean
+}> = ({ showTimer, history, active, isAtom }) => {
+  const ref = useRef<HTMLDivElement>(null)
+  return (
+    <LineChartWrapper showTimer={showTimer} ref={ref}>
+      <Svg>
+        <Path
+          stroke={active ? "#5f94f5" : "#737987"}
+          strokeOpacity={0.9}
+          strokeWidth={1}
+          fill="none"
+          d={history}
+        />
+        {!isAtom && (
```

Comment:
I'm not sure what `isAtom` is used for. I can only see it being passed to this component in one place, hardcoded as `true`.
I would have expected this component's render to be identical to the render of the original component, passing props where necessary.

Reply:
[Fixed here](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1970/commits/bebfc79ca887b2478f776dfaba36f2ce1f214d5c), for now I followed the structure we talked, but in this specific case it was different since the styleguide adds a dotted baseline svg, the `isAtom` prop was necesary in order to determine the length of this line (although that could be optimized too), for now I left the old version commented in case we need to get back there and we render the same as the original.


![Screenshot 2021-11-04 at 08 42 00](https://user-images.githubusercontent.com/11291319/140284433-01ec0bb6-213f-4e9d-b7c1-36246bd0d4f6.png)

Reply:
Ahh, understood. 
Maybe add the line back in but give the prop a more descriptive name so it is not tied to the context of styleguide.
Something like `showCenterLine` or similar.